README.md

```diff
@@ -1,13 +1,62 @@
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The following steps look similar, but with several options: You'll have a description of a scene, descriptions of how it looks in certain directions and the options to choose two of them. You also have the option to just go back.
+
+The game will count how many steps you need to get out of the maze. Even going back is counted as one step.
+
+As soon as you get out of the maze, the game ends and you'll have the option to restart the game by clicking a "Play again"-button. It will reload the page and you'll need to pick a new username to start again. 
+
+## The approach
+
+The game is built using React and Redux. Most of the functionality is stored in a reducer file called "game". There you will find reducers to 
+
+- enter a username
+- enter the game
+- take a step forward and
+- take a step back.
+
+The descriptions for the different steps are fetched from an API and the code for the fetches is stored in a thunk-file called "reusable". For the first step, the fetch will POST the chosen username to the API and get the first description back. In the following steps, the fetch POSTs the username, the type of action ("move") and the direction to get the next description back. 
+
+The Redux store will keep the current step as well as a history of the previous steps that is used for the "go back"-button. It also keeps track of how many steps have been taken. 
+
+The layout of the page was done using styled components and a grid. It contains of 
+- a header
+- some status info
+- the description for the scenery
+- buttons for the different directions
+- the descriptions for the different directions
+- the area for the "go back"-button and 
+- a footer. 
+
+In the center of the page you see a fox as a little avatar. The buttons to move forward will appear above, below or next to the fox depending on the direction, like on a compass. 
+
+There's a specific start page where you only see an input field and the "Start the game"-button. On the last page, you don't have any direction buttons or descriptions as you can't move further. Because the API-requests take some time, there is a loader animation showing between the steps. There is a firework animation at the final step.
+
+## Limitations
+
+So far, the game has no local storage. The information about your progress will be stored in the API and as soon as you reload the page and reenter the game with the same username, you'll start at the same point where you left. But the information about the steps you have taken earlier won't be accessible, so you don't have the chance to go back. 
+If there had been more time, we would have implemented a storage. We would also have liked to use animations for the avatar to make the user experience more fun. 
```

Comment:
As always, a wonderfully description readme. Love it.  
This section is more important than I can express.  Good job including it!